noir-projects/noir-contracts/contracts/ecdsa_public_key_note/src/lib.nr

```diff
@@ -1,30 +1,67 @@
-use dep::aztec::prelude::{NoteInterface, NullifiableNote, PrivateContext, RetrievedNote};
+use dep::aztec::prelude::{NullifiableNote, PrivateContext, RetrievedNote};
 
 use dep::aztec::{
     note::utils::compute_note_hash_for_nullify, keys::getters::{get_nsk_app, get_public_keys},
-    protocol_types::{address::AztecAddress, constants::{GENERATOR_INDEX__NOTE_NULLIFIER, GENERATOR_INDEX__NOTE_HASH}, hash::poseidon2_hash_with_separator, utils::arrays::array_concat},
-    macros::notes::note_custom_interface
+    protocol_types::{address::AztecAddress, constants::GENERATOR_INDEX__NOTE_NULLIFIER, hash::poseidon2_hash_with_separator, traits::Packable},
+    macros::notes::note
 };
 
-global ECDSA_PUBLIC_KEY_NOTE_LEN: u32 = 5;
-
 // Stores an ECDSA public key composed of two 32-byte elements
 // TODO: Do we need to include a nonce, in case we want to read/nullify/recreate with the same pubkey value?
-#[note_custom_interface]
+#[note]
```

Comment:
With the separation of the packing functionality from the NoteInterface it no longer made sense to use the `#[note_custom_interface]` here because the only custom thing here was the packing.